snap/snapcraft.yaml

```diff
@@ -41,3 +44,7 @@ plugs:
     interface: personal-files
     write:
       - $HOME/.local/share/juju
+  logs:
+    interface: personal-files
+    write:
+      - $HOME/.local/share/cou/log
```

Comment:
I think we should have write permission to `$HOME/.local/share/cou` not its subfolder